api/src/controllers/flows.ts

```diff
@@ -33,6 +33,10 @@ const webhookFlowHandler = asyncHandler(async (req, res, next) => {
 		}
 	);
 
+	if (preventCache) {
```

Comment:
Lets "Flip" the variables here. Lets make the vars and options "Cache Enabled" with "true" being the default, and then check against `if (cacheEnabled === false)` 👍🏻

Reply:
Great idea! Done~ 👍